nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -515,6 +517,7 @@ protected boolean commandRequiresNoGUI() {
             + "        nuxeoctl [mp-list|mp-listall|mp-init|mp-update] [command parameters] [-d [<categories>]|-q|--clid <arg>|--xml|--json]\n\n"
             + "        nuxeoctl [mp-reset|mp-purge|mp-hotfix|mp-upgrade] [command parameters] [-d [<categories>]|-q|--clid <arg>|--xml|--json|--accept <true|false|yes|no|ask>]\n\n"
             + "        nuxeoctl [mp-add|mp-install|mp-uninstall|mp-remove|mp-set|mp-request] [command parameters] [-d [<categories>]|-q|--clid <arg>|--xml|--json|--nodeps|--relax <true|false|yes|no|ask>|--accept <true|false|yes|no|ask>|-s|-im]\n\n"
+            + "        nuxeoctl [register-trial] <email> <company_name> [<project_name>]\n\n"
```

Comment:
square-brackets are wrong on those 10 commands, please remove them: the command is not optional

as said later, I would make email and company_name optional too (note that the other commands use camelcase or simple spaces but no underscores => companyName, projectName)

list your command in `org.nuxeo.launcher.NuxeoLauncher.OPTION_HELP_DESC_COMMANDS`